src/openrct2/ride/ShopItem.cpp

```diff
@@ -149,6 +149,33 @@ bool shop_item_has_common_price(const ShopItem shopItem)
     return (gSamePriceThroughoutPark & EnumToFlag(shopItem)) != 0;
 }
 
+bool shop_item_sold_elsewhere(const ShopItem shopItem, RideId excludeRideId)
+{
+    if (shopItem == ShopItem::None)
+        return false;
+
+    for (const auto& ride : GetRideManager())
+    {
+        if (ride.id == excludeRideId)
+            continue;
+
+        if (shopItem == ShopItem::Admission && ride.type == RIDE_TYPE_TOILETS)
```

Comment:
Don't add any RIDE_TYPE_* reference, we are currently trying to remove these from the code base.